Chapters/1-4-Chapter4.md

```diff
@@ -0,0 +1,232 @@
+
+My feet found some rest squatting and leaning against a tree near the tracks, hiding from the wind and from Kipnu if he had decided to follow me. _This is the right thing_, I affirmed to myself. _No externalities. No liabilities_. I thought about the crops I had just planted. Whether Melna would ask Tiggu to work on them. Whether she would kill them out of spite, or replant them in her own field. If this is as important as Jathi said, then those crops don't matter in the scheme of things. Heiko would understand.
+
+The station attendant called for us through a flathorn, "Train to Shaki approaching. All passengers prepare to marshal."
+
+I quickly picked up my bag and jumped the wall to the marshalling side. The attendant was tall for a maman with a recent flow tattoo mutating into hundreds of threads of infrared across hae neck. I admired it from afar as I shifted into the line for the 9th car. I had gotten a flow tattoo once, but working in the constant light of Ara made my skin's water vessels flow too quickly, so it all mixed together and just looked like I had some weird genetic defect in my hydroderm. My parents just laughed and told me about how they had tried it too when they were kids, and had the same results.
+
+The train came rumbling in with its engines and brakes humming with electrical blood. Thi called again, "Train to Shaki. Stops in Rudjan Ara, Rudjan Ulu, Yansha, Shaki Harbor, Shaki Central."
+
+The crowd fell into a tighter line against the wall, itching to find a decent place to sit.
+
+"Excuse me, would you help me get in?"
+
+Behind me was a older man with a polished steel cane that he was leaning on heavily. He looked up to me with a smile that I had to return, and I leant my hand to him to climb the steps into the carriage.
+
+"Thank you, my kaln," he said.
+
+I laughed, "I haven't been called a kaln in ages."
+
+"Oh, at my age, everyone is a youngling."
+
+_I suppose that's true._
+
+The seats were mostly empty but I took one across the carriage from the man - far enough not to creep him out - and put my bag down in the next seat. The light of Ara and the wind it brings had grown to its mid-day peak, but the reinforced curtains made them both much more pleasant for a long ride. The man was finding a good spot for his legs as the seats weren't exactly spacious enough for an extended leg, but he eventually settled on a position.
+
+"In all my years of riding trains, they've never built at least one spacious seat for people of my condition."
+
+"Maybe none of them are living with your problem," I said.
+
+"A sliver of empathy from them and an old man could sit in peace. Instead, a patient sweet maman like you has to bear my complaints."
+
+"Charming a young maman? At your age?"
+
+"I'm often told that I say too much of what I mean. Some may consider it charming, I suppose."
```

Comment:
Starting to lose track of who is saying what here. I know the advice is to write without dialog tags to make sure you nail down distinctive voices. But you should add them back in afterwards.